authentication/esignet-integration-impl/src/main/java/io/mosip/authentication/esignet/integration/dto/VciCredentialsDefinitionRequestDTO.java

```diff
@@ -0,0 +1,20 @@
+package io.mosip.authentication.esignet.integration.dto;
+
+import java.util.List;
+import java.util.Map;
+
+import lombok.Data;
+
+@Data
+public class VciCredentialsDefinitionRequestDTO {
```

Comment:
Rename this class to CredentialDefinitionDTO

Reply:
Done